linera-chain/src/block.rs

```diff
@@ -79,3 +79,23 @@ impl ConfirmedBlock {
         self.executed_block
     }
 }
+
+use linera_base::{data_types::BlockHeight, identifiers::ChainId};
+use linera_execution::committee::Epoch;
```

Comment:
These should be with the other imports.

Reply:
Right. I left them here kinda on purpose - don't you think that `Epoch` should be with the other two? i.e. under same module? Don't they all belong to the same "group" of data?

Reply:
Yes, it's a bit weird that that's under `linera_execution::committee`. But maybe that's also something for a different PR?

Reply:
Yes, definitely. I kept the imports here so that I don't forget asking about it. I will move them up with the rest.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/2845/commits/4b2dc5a6b6f4d18c3d63d6970770ed231885bade